grails-app/controllers/swagger/grails/UrlMappings.groovy

```diff
@@ -2,6 +2,6 @@ package swagger.grails
 
 class UrlMappings {
     static mappings = {
-        "/"(controller: "swagger", action: "api")
+        "/swagger"(controller: "swagger", action: "api")
```

Comment:
since this has already been published with the controller mapped to `"/"` i think its best to leave it that way.